plugins/extrusions/extrusions.py

```diff
@@ -0,0 +1,146 @@
+from typing import TypeVar, Union, Tuple
+
+# import importlib.resources
+import pkg_resources
+import cadquery as cq
+from cadquery.occ_impl import importers
+from cadquery.occ_impl.geom import Vector, Location
```

Comment:
Why did you use the lower level Vector and Location classes? They are also available on the cadquery module. Examples [here](https://cadquery.readthedocs.io/en/latest/classreference.html?highlight=Vector#cadquery.Vector) and [here](https://cadquery.readthedocs.io/en/latest/classreference.html?highlight=location#cadquery.Location). Are the higher level implementations missing something that you needed?

Reply:
I wasn't aware of the higher level implementations - I'm quite new to the cadquery code. I'll update.